UI/Pages/TuitionPartner.cshtml

```diff
@@ -132,31 +132,22 @@
 		    <thead class="govuk-table__head">
```

Comment:
I believe there's some additional content that goes above the table according to the ticket:

```
<p>The costs shown are per pupil per hour. 
[Next line] All subjects are the same price but depend on tuition group size.
```

However the second line talks about subjects having the same price which wouldn't work for the ones that do vary by subject. Can you go back to the BAs/content and ask what should be displayed when the price does vary by subject?

Reply:
Yep, I missed that.  I have added the extra content.  

I couldn't find any examples of current tuition partners that varied the price by subject, but I will ask the question.

There is a new ticket for showing only the tuition types available in the searched for postcode: https://dfedigital.atlassian.net/browse/NTP-451

Reply:
Nice one. Reed Tutors /tuition-partner/reed-tutors are the best example of a TP that varies price by subject